app/labor/badge_rewarder.rb

```diff
@@ -20,7 +20,7 @@ def self.award_beloved_comment_badges
     # ID 3 is the proper ID in prod. We should change in future to ENV var.
     badge_id = Badge.find_by(slug: "beloved-comment")&.id || 3
     Comment.where("positive_reactions_count > ?", 24).find_each do |comment|
-      message = "You're DEV famous! [This is the comment](https://dev.to#{comment.path}) for which you are being recognized. 😄"
+      message = "You're DEV famous! [This is the comment](https://#{ApplicationConfig['APP_DOMAIN']}#{comment.path}) for which you are being recognized. 😄"
```

Comment:
should we rename this to `famous` instead of `DEV famous` ? Or use `COMMUNITY_NAME` here too?